reledmac.dtx

```diff
@@ -7107,7 +7107,7 @@
 % 
 % \begin{macro}{\@beforeinsertofthisedtext}
 % \cs{@beforeinsertofthisedtext} is an internal macro. \macpackage or \parpackage can add in this macro any content required to be executed before doing any \cs{insert} related to a \cs{edtext}.
-% It content is let to \cs{relax} at the end of every \cs{edtext}.
+% Its content is added to \cs{relax} at the end of every \cs{edtext}.
```

Comment:
Dear Maïeul,

I’m trying to learn how to use the Github desktop application. In order to to do, I made small English language changes to two files and - if I did it right - issued a pull request to you for a branch called “marks-experiment”. You need not commit the changes, since they are pretty unimportant.

However, could you let me know if I did the right things the right way?

Thanks!

Mark